code/game/machinery/spaceheater.dm

```diff
@@ -179,7 +179,7 @@
 	heating_energy = laser * BASE_HEATING_ENERGY
 
 	settable_temperature_range = cap * 30
-	efficiency = (cap + 1) * 10
+	efficiency = (cap + 1) * 100
```

Comment:
```suggestion
	efficiency = (cap + 1) * initial(efficiency) * 0.5
```
Now we won't have to change this proc when we change the efficiency. Could do the same for the other variables that don't use defines.

Reply:
Added it for this and settable temp range